tests/shared/core/training_data/test_structures.py

```diff
@@ -56,37 +56,6 @@ def test_session_start_is_not_serialised(domain: Domain):
     assert actual == expected
 
 
-def test_as_story_string_or_statement():
-    import rasa.shared.utils.io
-
-    stories = """
-    stories:
-    - story: hello world
-      steps:
-      - or:
-        - intent: intent1
-        - intent: intent2
-        - intent: intent3
-      - action: some_action
-    """
-
-    reader = YAMLStoryReader(is_used_for_training=False)
-    yaml_content = rasa.shared.utils.io.read_yaml(stories)
-
-    steps = reader.read_from_parsed_yaml(yaml_content)
-
-    assert len(steps) == 1
-
-    assert (
-        steps[0].as_story_string()
```

Comment:
Is this no longer used?

Reply:
It isn't. `is_used_for_training=False` can no longer be used so the tracker(s) generated by `StoryReader` can only be used for training.

Reply:
So can we delete `as_story_string()` as part of removing mark down support?

Reply:
It still seems to be used by the interactive learning so I would skip that for now.

Reply:
So should we keep the test then?